labelbox/client.py

```diff
@@ -533,3 +533,20 @@ def get_models(self, where=None):
             An iterable of Models (typically a PaginatedCollection).
         """
         return self._get_all(Model, where, filter_deleted=False)
+
+    def create_model(self, **kwargs):
+        """ Creates a Model object on the server.
+
+        Attribute values are passed as keyword arguments.
+
+        >>> model = client.create_model(name="<model_name>", ontology_id="<ontology_id>")
```

Comment:
maybe just make name and ontology_id required args

Reply:
Done